api/policy.go

```diff
@@ -90,6 +93,36 @@ type PolicyUpsert struct {
 	Payload string `jsonapi:"attr,payload"`
 }
 
+// PolicyPatch is the message to patch a policy.
+type PolicyPatch struct {
+	ID int
+
+	// Standard fields
+	// Value is assigned from the jwt subject field passed by the client.
+	// CreatorID is the ID of the creator.
+	UpdaterID int
+	RowStatus *string `jsonapi:"attr,rowStatus"`
+
+	// Related fields
+	EnvironmentID *int `jsonapi:"attr,environmentId"`
+
+	// Domain specific fields
+	Type    PolicyType
```

Comment:
Needs comment why we need the type, since ID is enough to identify the policy.

Same for delete

Reply:
Comments updated.